service/src/main/java/bio/terra/pipelines/stairway/imputation/CheckCbasHealthStep.java

```diff
@@ -36,8 +35,7 @@ public StepResult doStep(FlightContext flightContext)
         cbasService.checkHealth(cbasUri, samService.getTspsServiceAccountToken());
     if (!healthResult.isOk()) {
       return new StepResult(
-          StepStatus.STEP_RESULT_FAILURE_RETRY,
-          new CbasServiceApiException(new ApiException("CBAS is not healthy")));
```

Comment:
it was silly that I committed this in the first place, I shouldn't need to make a whole ApiException object when all i want to pass is a string.  This feels much cleaner